docs/source/extend_kedro/plugins.md

```diff
@@ -44,6 +44,32 @@ Once the plugin is installed, you can run it as follows:
 kedro to_json
 ```
 
+## Extending Starters Template
+You can extend `kedro starter list` and `kedro new` by introducing your own `cookiecutter` template.
```

Comment:
```suggestion
It is possible to extend the list of starter aliases built into Kedro. This means that a [custom Kedro starter](create_kedro_starters.md) can be used directly through the `starter` argument in `kedro new` rather than needing to explicitly provide the `template` and `directory` arguments. A custom starter alias behaves in the same way as an official Kedro starter alias and is also picked up by `kedro starter list`. 
```

Also would be good to link something on the "create kedro starters" page point to this new section. Maybe a `{note}` at the end saying you can add an alias by creating a plugin using `kedro.starters` entrypoint, linking to here.